src/accordion/block.js

```diff
@@ -102,6 +103,14 @@ registerBlockType( 'purdue-blocks/accordion', {
               } }
             />
           </PanelRow>
+          <PanelRow>
+              <TextControl
+                label="HTML Anchor"
+                help="Enter a word without spaces to make a unique web address just for this block, called an “anchor.” Then, you’ll be able to link directly to this section of your page."
```

Comment:
Probably specify more clearly here that the anchor needs to be unique from any other anchors on the page.